src/lib.rs.in

```diff
@@ -802,6 +802,29 @@ class DocumentAccessResult {
     method DocumentAccessChange::errors(&self) -> FailedResult; alias getErrors;
 });
 
+foreigner_class!(
+    class PolicyCachingConfig{
+        self_type PolicyCachingConfig;
+        constructor policy_caching_config::create(max_entries: usize) -> PolicyCachingConfig;
```

Comment:
I think we should provide a `default` fn here, or some other mechanism for callers to just take the default.

Reply:
Good call, done.